Python/Product/PythonTools/PythonTools/LanguageServerClient/FileWatcher/Listener.cs

```diff
@@ -108,72 +108,91 @@ class Listener : IDisposable {
             await OnFileChanged(sender, e);
         }
         private async Task OnFileChanged(object sender, System.IO.FileSystemEventArgs e) {
-            // Skip directory change events and when rpc has ben disconnected
-            if (!e.IsDirectoryChanged() && _rpc != null) {
```

Comment:
Ugh sorry about the formatting showing up as one big change. I should have put in the logic change as its own commit, then the early returns as another one so it would be easier to see the diff.

The only changes are:

1. Make sure the _rpc isn't null when you try to use it
2. Catch all ConnectionLostExceptions from _rpc calls and ignore them.